tests/v1/test_upcoming.py

```diff
@@ -6,6 +6,28 @@
 
 
 def test_get_upcoming_changes(client, api_prefix):
+    response = client.get(f"{api_prefix}/upcoming-changes")
+    assert response.status_code == 200
+    data = response.json()["data"]
+    assert data[0]["name"] == "Add Node.js to RHEL9 AppStream THIS IS TEST DATA"
+    for d in data:
+        assert "potentiallyAffectedSystems" not in d["details"]
+        assert "potentiallyAffectedSystemsCount" not in d["details"]
```

Comment:
Could of nits:

Group setup and assertions instead of intermingling them to make tests easier to read.

Instead of `for` loops, try to break up assertions into statements with explanations. It will make figuring out the purpose of the test and why it failed easier in the future.
```suggestion
    response = client.get(f"{api_prefix}/upcoming-changes")
    data = response.json()["data"]
    
    assert response.status_code == 200
    assert data[0]["name"] == "Add Node.js to RHEL9 AppStream THIS IS TEST DATA"
    assert not any("potentiallyAffectedSystems" in d["details"] for d in data), "Why did this fail?"
    assert not any("potentiallyAffectedSystemsCount" in d["details"] for d in data), "Why did this fail?"
```

Reply:
fixed